common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -361,6 +361,91 @@ def test_showanswer_past_due(self):
                                             graceperiod=self.two_day_delta_str)
         self.assertFalse(still_in_grace.answer_available())
 
+    def test_showanswer_after_attempts_with_max(self):
```

Comment:
Admittedly, this is one area where I'm willing to be _slightly_ more lenient...
_if_ _and_ _only_ _if_ the test name itself is sufficiently expressive.

When there is no docstring, the function name can be used instead;
_Sometimes_ that suffices.

Consider:

```python
def test_show_answer_after_max_attempts(self):
    """
    Test that show answer functions properly after the max attempts has been reached
    """
    pass
```

Here, the comment just rewords the test name, so I would argue the comment is not necessary.

On the other hand:

```python
def test_show_answer_after_max_attempts_but_only_if_we_have_configured_some_certain_values_which_may_affect_this(self):
    pass
```

That's far too wordy for a method name, so the method name should be shortened and the special case explained via docstring.

Here though, I agree with @caseylitton; this could be expressed a bit more clearly.